src/Build.OM.UnitTests/Construction/ProjectSdkImplicitImport_Tests.cs

```diff
@@ -398,5 +452,11 @@ private void VerifyPropertyFromImplicitImport(Project project, string propertyNa
 
             Assert.Equal(expectedValue, property.EvaluatedValue);
         }
+
+        private SdkReference GetParsedSdk(ProjectImportElement element)
+        {
+            PropertyInfo parsedSdkInfo = typeof(ProjectImportElement).GetProperty("ParsedSdkReference", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Isn't the point of this test assembly to _not_ have to depend on internals?

Reply:
Yeah, but for verification I thought this was cleaner than anything else. I was concerned mainly with parsing, and using this allows me to assert that the data going to the resolver from the evaluator is exactly what was intended. It might mean there's not enough disambiguation in the OM since I think some of the internals have changed since it was originally exposed. But not something that I wanted to change at this point.